crates/pop-parachains/src/templates.rs

```diff
@@ -123,19 +123,34 @@ pub enum Parachain {
 	EVM,
 	// OpenZeppelin
 	#[strum(
-		serialize = "polkadot-generic-runtime-template",
+		serialize = "generic-template",
 		message = "Generic Runtime Template",
-		detailed_message = "A generic template for Substrate Runtime",
+		detailed_message = "A generic template for Substrate Runtime.",
 		props(
 			Provider = "OpenZeppelin",
 			Repository = "https://github.com/OpenZeppelin/polkadot-runtime-templates",
 			Network = "./zombienet-config/devnet.toml",
-			SupportedVersions = "v1.0.0",
+			SupportedVersions = "v1.0.0,v2.0.1",
 			IsAudited = "true",
 			License = "GPL-3.0"
 		)
 	)]
 	OpenZeppelinGeneric,
+	// OpenZeppelin EVM
+	#[strum(
+		serialize = "evm-template",
```

Comment:
Whilst this is done for ease of use in locating the directory within the repo, in terms of the list of template names there is nothing here to disambiguate it from an evm template from any other provider, potentially resulting in a naming clash. As it is a new template it is an opportunity to specify something more useful.

Reply:
Renamed to `openzeppelin/evm-template` and introduced a `template_name_without_provider` method, which will also be useful for Tanssi integrations too.